translations/mod-fqm-manager/en.json

```diff
@@ -0,0 +1,181 @@
+  "entityType.drv_item_details.item_effective_location_name": "Item effective location name",
+  "entityType.drv_item_details.item_hrid": "Item hrid",
+  "entityType.drv_item_details.id": "Item ID",
+  "entityType.drv_item_details.item_material_type": "Item material type",
+  "entityType.drv_item_details.item_material_type_id": "Item material ID",
+  "entityType.drv_item_details.item_permanent_location_id": "Item permanent ID",
+  "entityType.drv_item_details.item_permanent_location_name": "Item permanent location name",
+  "entityType.drv_item_details.item_status": "Item status",
+  "entityType.drv_item_details.item_temporary_location_id": "Item temporary ID",
+  "entityType.drv_item_details.item_temporary_location_name": "Item temporary location name",
+  "entityType.drv_item_details.item_updated_date": "Item updated date",
+
+  "entityType.drv_item_callnumber_location": "Item, Call number and location",
+  "entityType.drv_item_callnumber_location.id": "Item ID",
+  "entityType.drv_item_callnumber_location.item_hrid": "Item hrid",
+  "entityType.drv_item_callnumber_location.item_effective_call_number": "Effective call number",
+  "entityType.drv_item_callnumber_location.item_effective_call_number_typeid": "Effective call number typeId",
+  "entityType.drv_item_callnumber_location.item_effective_call_number_type_name": "Effective call number type name",
+  "entityType.drv_item_callnumber_location.item_level_call_number": "Call number",
+  "entityType.drv_item_callnumber_location.item_level_call_number_typeid": "Call number typeId",
+  "entityType.drv_item_callnumber_location.item_level_call_number_type_name": "Call number type name",
+  "entityType.drv_item_callnumber_location.item_holdings_record_id": "Holdings record ID",
+  "entityType.drv_item_callnumber_location.item_status": "Item status",
+  "entityType.drv_item_callnumber_location.item_copy_number": "Item copy number",
+  "entityType.drv_item_callnumber_location.item_barcode": "Item barcode",
+  "entityType.drv_item_callnumber_location.item_created_date": "Item created date",
+  "entityType.drv_item_callnumber_location.item_updated_date": "Item updated date",
+  "entityType.drv_item_callnumber_location.item_effective_location_id": "Item location ID",
+  "entityType.drv_item_callnumber_location.item_effective_location_name": "Item location name",
+  "entityType.drv_item_callnumber_location.item_effective_library_id": "Item effective library id",
+  "entityType.drv_item_callnumber_location.item_effective_library_name": "Item effective library name",
+  "entityType.drv_item_callnumber_location.item_effective_library_code": "Item effective library code",
+  "entityType.drv_item_callnumber_location.item_permanent_location_id": "Item permanent ID",
+  "entityType.drv_item_callnumber_location.item_permanent_location_name": "Item permanent location name",
+  "entityType.drv_item_callnumber_location.item_material_type_id": "Material ID",
```

Comment:
Should it be Item material Type ID like we have in definition in from of label alias?

Reply:
I pulled these from the current DB; most recent one I saw was 
![image](https://github.com/folio-org/mod-fqm-manager/assets/8005215/aa3b5a52-2428-4d22-9326-f4c672b2a704) (in v1.0.2 `update-items-holdingsrecord-entity-type-definition`)

Reply:
There are inconsistencies across the entity type names; these were pulled directly from the original views.  See [MODFQMMGR-88](https://issues.folio.org/browse/MODFQMMGR-88).